desktop/troubleshoot/known-issues.md

```diff
@@ -1,20 +1,22 @@
 ---
-description: Known issues for Mac
+description: Find known issues for Docker Desktop
 keywords: mac, troubleshooting, known issues, Docker Desktop
-title: Known issues for Docker Desktop on Mac
+title: Known issues
 ---
 <ul class="nav nav-tabs">
-  <li class="active"><a data-toggle="tab" data-target="#tab3">Known issues for Mac with Intel chip</a></li>
-  <li><a data-toggle="tab" data-target="#tab4">Known issues for Mac with Apple silicon</a></li>
+  <li class="active"><a data-toggle="tab" data-target="#tab2">For all platforms</a></li>
+  <li><a data-toggle="tab" data-target="#tab3">For Mac with Intel chip</a></li>
+  <li><a data-toggle="tab" data-target="#tab4">For Mac with Apple silicon</a></li>
 </ul>
 <div class="tab-content">
+<div id="tab2" class="tab-pane fade" markdown="1">
```

Comment:
I think this tab body needs the `in active` classes, and it should be removed from the mac with intel tab.

```suggestion
<div id="tab2" class="tab-pane fade in active" markdown="1">
```